### PR TITLE
Crawler fixing

### PR DESCRIPTION
**Pull request by devtrap**
1. Fix crawler to get snapshot from files recursively 
2. Fix unit test result for google_template_processor 
3. Fix unit test result for azure_template_processor
4. Fix unit test result for aws_template_processor
5. Remove unused variable from base_template_processor

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -18,7 +18,7 @@
 setup(
     name='prancer-basic',
     # also update the version in processor.__init__.py file
-    version='1.0.36',
+    version='1.0.37',
```
Comment (ajeybk) [inline setup.py]: Make it 1.0.38 as 1.0.37 is released.

**Review by ajeybk — COMMENTED** [review]
Fix the version, rest looks good.

File: tests/processor/template_processor/aws/test_aws_template_processor.py [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import pytest
 import os
+import json
```
Comment (vatsalgit5118) [inline tests/processor/template_processor/aws/test_aws_template_processor.py]: Remove unused import

File: setup.py (hunk 1 above)
Comment (vatsalgit5118) [reply setup.py]: Changed Version to 1.0.38

File: tests/processor/template_processor/aws/test_aws_template_processor.py (hunk 2 above)
Comment (vatsalgit5118) [reply tests/processor/template_processor/aws/test_aws_template_processor.py]: Removed extra import

**Review by vatsalgit5118 — APPROVED** [review]

File: setup.py (hunk 1 above)
Comment (ajeybk) [reply setup.py]: ok

**Review by ajeybk — APPROVED** [review]
Looks good

**Pull request merged**